examples/azure/service-account/values.yaml

```diff
@@ -0,0 +1,5 @@
+serviceAccount:
+  create: true
+  name: graphdb
```

Comment:
The `name` field is not really required, the chart will generate one based on the release name and templates.